src/snowflake/cli/_plugins/git/commands.py

```diff
@@ -37,6 +36,7 @@
     like_option,
 )
 from snowflake.cli.api.commands.snow_typer import SnowTyperFactory
+from snowflake.cli.api.commands.structs import OnErrorType
```

Comment:
nit: not sure about `structs` as a name. `types` or `common` are more in line with existing names IMHO.

Reply:
Changed to `common`